braindecode/datasets/xy.py

```diff
@@ -14,7 +14,7 @@
 
 
 def create_from_X_y(
-        X, y, drop_last_window, sfreq=None, ch_names=None, window_size_samples=None,
+        X, y, sfreq, drop_last_window, ch_names=None, window_size_samples=None,
```

Comment:
this is an API change.

@robintibor deprecation cycle or you can live with it?